### ISSUE TITLE
Create neuralnetworks.py class

### ISSUE THREAD

**Issue by tjhollan** [tjhollan/ReinforcementLearning#2]
## Neural Network
The Neural Network will require the following methods:
-  `__init__(n_inputs, [n_hiddens_per_layer], n_outputs)` - `n_inputs` is the number of inputs to the neural network, `[n_hiddens_per_layer]` is a list of the size of each hidden layer, `n_outputs` is the number of output values for the network
-  `use(X)` - takes one argument: the state and action pair; returns a rating between -1 and 1
-  `train(X, T, epochs, learning_rate)` - takes four arguments: the state/action pair as an array, the target, the number of epochs for which to train, and the learning rate; updates internal weights

We must be able to initialize the network with a number of inputs, hidden layers, outputs, and toggle standardizing the inputs off (somehow).
